src/docfx/lib/html/HtmlToken.cs

```diff
@@ -15,7 +15,7 @@ internal struct HtmlToken
 
         public ReadOnlyMemory<char> RawText { get; }
 
-        public Memory<HtmlAttribute> Attributes { get; }
+        public Memory<HtmlAttribute> Attributes { get; private set; }
```

Comment:
how about maintaining a field `private HtmlAttribute[] _attributes` and return `_attributes.AsMemory()` for this readonly property.